CHANGES.md

```diff
@@ -1,5 +1,29 @@
 # Changes
 
+## edge-21.6.2
+
+This release fixes a problem with the HTTP body buffering that was added
+to support gRPC retries. Now, Only requests with a retry configuration
+are buffered (and only when their bodies are less than 64KB).
+Additionally, An Issue where forwarded HTTP traffic could fail to
+detect when the target pod was deleted, causing connections to forever
+has been fixed. This only impacted traffic forwarded directly to pod IPs
+(and not load balanced services)
+
+Finally, This release also includes some fixes in the CLI and dashboard.
+
+* Added a new check that verifies if the opaque ports annotation is
+  misconfigured on services or pods (thanks @migue!)
+* Added support for resource aware completion for core linkerd command
+* Fixed an issue where `namespace` resource was also being shown
```

Comment:
Do you think it'd be valuable here to be explicit that this was done in the dashboard/web component? 

Also, quite nitpicky but I'd perhaps say:
```suggestion
* Fixed an issue where `namespace` resource was erroneously being shown
```